samples/calendar/index.js

```diff
@@ -0,0 +1,143 @@
+
+
+const fs = require('fs');
+const readline = require('readline');
```

Comment:
So for most of the OAuth2 workflow examples, we use this `sampleClient.js` thingy that abstracts most of the goo away.  Could you check out the drive sample, and make it mostly match that approach?  It's nice, because it hides away auth for the individual interesting sample you're adding. 
https://github.com/googleapis/google-api-nodejs-client/blob/master/samples/drive/list.js